Retired fix1543 amendment

pratikmankawde opened the pull request:
## High Level Overview of Change
This change retires the `fix1543` amendment.

### Context of Change
Amendments activated for more than 2 years can be retired.
https://xrpl.org/resources/known-amendments#fix1543

### Type of Change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Refactor (non-breaking change that only restructures code)
- [ ] Performance (increase or change in throughput and/or latency)
- [ ] Tests (you added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation update
- [ ] Chore (no impact to binary, e.g. `.gitignore`, formatting, dropping support for older tooling)
- [ ] Release

bthomee commented:
@pratikmankawde there's a conflict - can you please resolve it first? I'd also like to get a sign-off from @a1q123456 before merging.

a1q123456 commented on src/xrpld/app/tx/detail/Escrow.cpp:
```diff
@@ -1226,10 +1224,9 @@ EscrowFinish::doApply()
 
 std::uint32_t
-EscrowCancel::getFlagsMask(PreflightContext const& ctx)
+EscrowCancel::getFlagsMask(PreflightContext const&)
```
The default implementation for this method is already the same. I think we can remove this function.
See src/xrpld/app/tx/detail/Transactor.cpp

pratikmankawde replied:
Yeah, makes sense.

a1q123456 commented on src/xrpld/app/tx/detail/PayChan.cpp:
```diff
@@ -336,10 +335,9 @@ PayChanFund::makeTxConsequences(PreflightContext const& ctx)
 
 std::uint32_t
-PayChanFund::getFlagsMask(PreflightContext const& ctx)
+PayChanFund::getFlagsMask(PreflightContext const&)
```
Same as above

pratikmankawde replied:
Done

a1q123456 commented on src/xrpld/app/tx/detail/Escrow.cpp:
```diff
@@ -640,10 +639,9 @@ EscrowFinish::checkExtraFeatures(PreflightContext const& ctx)
 
 std::uint32_t
-EscrowFinish::getFlagsMask(PreflightContext const& ctx)
+EscrowFinish::getFlagsMask(PreflightContext const&)
```
Same as `EscrowCancel::getFlagsMask`

pratikmankawde replied:
Done

a1q123456 commented on src/xrpld/app/tx/detail/Escrow.cpp:
```diff
@@ -119,10 +119,9 @@ escrowCreatePreflightHelper<MPTIssue>(PreflightContext const& ctx)
 
 std::uint32_t
-EscrowCreate::getFlagsMask(PreflightContext const& ctx)
+EscrowCreate::getFlagsMask(PreflightContext const&)
```
Same as EscrowCancel::getFlagsMask

pratikmankawde replied:
Done

pratikmankawde commented:
> @pratikmankawde there's a conflict - can you please resolve it first? I'd also like to get a sign-off from @a1q123456 before merging.

Done

bthomee commented:
@pratikmankawde There are some linking errors due to removing some of the `getFlagsMask` overrides - I'm sure it's an easy fix.

a1q123456 reviewed (approved):
lgtm

bthomee commented:
@pratikmankawde Ready to merge?

pratikmankawde commented:
> @pratikmankawde Ready to merge?

Yeah.

bthomee committed:
refactor: Retire fix1543 amendment

Amendments activated for more than 2 years can be retired. This change retires the fix1543 amendment.
